plugins/plugins/commands

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+case "$1" in
+  plugins-install)
```

Comment:
If we did split these out the command would be `dokku plugins:plugins-install` and I didn't necessarily want break out these simpler commands and potentially add more confusion amongst the user base? I don't know that I have a strong stance against breaking them out though.